src/js/popup.js

```diff
@@ -87,44 +95,53 @@ class UI {
     return paperId;
   }
 
-  setFormContents(paperTitle, abst, authors) {
-    document.getElementById("js-title").value = paperTitle;
-    document.getElementById("js-abst").value = abst;
+  setFormContents(paperTitle, abst, comment, authors) {
+    document.getElementById('js-title').value = paperTitle;
+    document.getElementById('js-abst').value = abst;
+    //     document.getElementById('js-published').value = published;
+    document.getElementById('js-comment').value = comment;
     authors.forEach((author) => {
-      console.log(author);
       const template = `<span class="uk-badge uk-margin-small-right uk-margin-small-top">{{ text }}</span>`;
       const rendered = Mustache.render(template, { text: author });
       document
-        .getElementById("js-chip-container")
-        .insertAdjacentHTML("beforeend", rendered);
+        .getElementById('js-chip-container')
+        .insertAdjacentHTML('beforeend', rendered);
     });
   }
 
   async getArXivInfo(url) {
     this.showProgressBar();
     const paperId = this.parseArXivId(url);
 
-    const res = await fetch(ARXIV_API + "?id_list=" + paperId.toString());
+    const res = await fetch(ARXIV_API + '?id_list=' + paperId.toString());
     if (res.status != 200) {
-      console.log("[ERR] arXiv API request failed");
+      console.log('[ERR] arXiv API request failed');
       return;
     }
     const data = await res.text(); // TODO: error handling
     console.log(res.status);
-    const xmlData = new window.DOMParser().parseFromString(data, "text/xml");
+    const xmlData = new window.DOMParser().parseFromString(data, 'text/xml');
     console.log(xmlData);
 
-    const entry = xmlData.querySelector("entry");
-    const paperTitle = entry.querySelector("title").textContent;
-    const abst = entry.querySelector("summary").textContent;
-    const authors = Array.from(entry.querySelectorAll("author")).map(
+    const entry = xmlData.querySelector('entry');
+    const paperTitle = entry.querySelector('title').textContent;
+    const abst = entry.querySelector('summary').textContent;
+    const authors = Array.from(entry.querySelectorAll('author')).map(
       (author) => {
         return author.textContent.trim();
       }
     );
-
-    this.setFormContents(paperTitle, abst, authors);
-    return { title: paperTitle, abst: abst, authors: authors, url: url };
+    const published = entry.querySelector('published').textContent;
+    const comment = entry.querySelector('comment')?.textContent ?? 'none';
```

Comment:
refactored: https://github.com/wangjksjtu/arxiv2notionplus/blob/main/src/js/popup.js#L129-L136

we can use ``??'' to write the same process in a single line.